fr/user/articles/README.md

```diff
@@ -1,11 +1,11 @@
 Articles
 ========
-Articles are the core feature of wallabag. wallabag's content parser, named Graby gets the content from articles, and only the content.
+Les articles sont une fonctionalités clé dans wallabag. Graby, le *parser* de contenus de wallabag récupère les contenus des articles, et uniquement le contenu.
 
-## Mark articles as read
+## Marquer les articles comme « lu »
 
-When you mark an article as read, it gets out of the category « Unread » and gets puts in the « Archive » category.
+Lorsque vous marquez un article comme « lu », il est déplacé depuis la catégorie « Non lus » vers la catégorie « Archive »
```

Comment:
The french translation of "Archive" in the menu is "lus", see https://hosted.weblate.org/translate/wallabag/messages/fr/?checksum=3d0e5c67d5bae135&q=archive&sort_by=-priority%2Cposition